frontend/controller/service-worker.js

```diff
@@ -285,24 +244,12 @@ sbp('sbp/selectors/register', {
           console.info(`[service-worker/setup-push-subscription] got ${newSub ? 'new' : 'existing'} subscription '${subID}':`, endpoint)
         }
         console.info('[service-worker/setup-push-subscription] calling push/reportExistingSubscription on:', subID)
-        await sbp('push/reportExistingSubscription', subscription?.toJSON())
+        await sbp('push/reportExistingSubscription', subscription?.toJSON(), subscription?.options.applicationServerKey)
       } catch (e) {
         console.error('[service-worker/setup-push-subscription] error getting a subscription:', e)
-        if (e?.message === 'WebSocket connection is not open') {
```

Comment:
Do you remember the reasoning for choosing to retry on all errors?

Reply:
https://okturtles.slack.com/archives/C0EH7P20Y/p1741305585842639?thread_ts=1741276573.013899&cid=C0EH7P20Y